recipes/cpython/all/conanfile.py

```diff
@@ -173,6 +183,9 @@ def validate(self):
                 if self.dependencies["mpdecimal"].ref.version < Version("2.5.0"):
                     raise ConanInvalidConfiguration("cpython 3.9.0 (and newer) requires (at least) mpdecimal 2.5.0")
 
+        if self.settings.compiler == "gcc" and Version(self.settings.compiler.version).major == 9 and Version(self.version) >= "3.12":
```

Comment:
Could you please point the error?

Reply:
https://c3i.jfrog.io/c3i/misc/logs/pr/22599/17-linux-gcc-legacy/cpython/3.12.2//2a6be3fe8381eeefd48805e3669056270fb8f4a1-build.txt - search for "posixshmem". Locally I got an internal compiler error, not entirely sure what the error in CCI is, but I do know that it's GCC 9 exclusive (or maybe it affects 8 and 10, but not sure since those aren't tested in CCI)

Reply:
Thank you, I'll check if is something exclusive with docker image, due old ubuntu version, or is gcc9. Depending the case, we can skip the build, but allow users to build with gcc9

Reply:
I also get an ICE locally with GCC 9, so my guess is that it's just that version.